README.md

```diff
@@ -60,7 +60,7 @@ are tentative.
 
 HDF4 4.3.0 will clean up some long-standing issues in the library
 * The problematic XDR library will be replaced with a thin I/O layer that correctly handles long integer data on all platforms
-* We will drop support for FORTRAN 77 so the wrappers can be built like they are in HDF5
+* We will drop support for FORTRAN 77 so the wrappers can be built like they are in HDF4
```

Comment:
Is it supposed to be: "...like they are in HDF5"?

Reply:
IDK, I just did a global S & R because of other locations.

Reply:
updated